lamp/tasks/install_php.yml

```diff
@@ -16,7 +16,7 @@
   apt:
       name: "php{{ '' if php_version == '7.0' else php_version }}-{{ item }}"
       state: present
-  with_items: "{{ ['fpm', 'cli', 'mysql'] + php_modules | default([]) }}"
+  with_items: "{{ ['fpm', 'cli', 'mysql', 'zip'] + php_modules | default([]) }}"
```

Comment:
Scratch that — for 7.0, I was looking at a project that explicitly installed `php-zip`. I’m OK with including it by default here to prevent Composer issues in all cases.